jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -423,29 +430,38 @@ static String getServerName(Request request)
         return local == null ? null : local.toString();
```

Comment:
Is it not this logic the same as `ConnectionMetaData getServerAuthority(HttpConfiguration httpConfiguration, ConnectionMetaData connectionMetaData)`?

I would call that, and then return `hostPort.getHost()`, so there is only 1 logic.